control-plane/roles/gardener/tasks/main.yaml

```diff
@@ -55,6 +55,7 @@
       - gardener_dns_provider is not none
       - gardener_cloud_profile_metal_api_url is not none
       - gardener_cloud_profile_metal_api_hmac is not none
+      - gardener_cert_management_issuer_email is not none
```

Comment:
Yes, place is now correct, however when checking `is not none` you should add `gardener_cert_management_issuer_email:` to the defaults. Otherwise Ansible will fail with "variable not defined" but not with the assertion.

It would also be possible to check for `is defined`. But I think first variant is little better.